### PR TITLE
LPS-127037

### PR DESCRIPTION
**Pull request by martamedio**
See comments on: https://issues.liferay.com/browse/LPS-127037
cc/ @linolaoj @gvasconcelos @lipusz 

Please @csierra, take a look at my changes too

Thanks!

### CODE REVIEW

**Comment by martamedio** [comment]
ci:test:sf

File: modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -20,6 +20,8 @@ provider-name-help=Set the name for the OpenID Connect provider.
 scopes-help=Set the scopes Liferay will request during authentication. Scopes are delimited with spaces.
```
Comment (liferay-appsec) [inline modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/resources/content/Language.properties:20]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by martamedio** [comment]
Please, see:
- https://github.com/liferay-appsec/liferay-portal/pull/247
- https://github.com/liferay-appsec/liferay-portal/pull/256

**Pull request closed without merging**